scripts/release_scripts/update_changelog_and_credits_test.py

```diff
@@ -644,6 +671,9 @@ def mock_create_branch(
                 unused_repo, unused_repo_fork, unused_target_branch,
                 unused_github_username, unused_current_release_version_number):
             check_function_calls['create_branch_gets_called'] = True
+        def mock_inform_server_errors_team(
+                unused_release_rota_url, unused_server_error_playbook_url):
```

Comment:
```suggestion
        def mock_inform_server_errors_team(
            unused_release_rota_url, unused_server_error_playbook_url
        ):
```

(this is more easier to type later, as there is more space for the return type)

Reply:
Done.